x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/bulk_actions/use_bulk_actions.tsx

```diff
@@ -53,19 +57,25 @@ interface UseBulkActionsArgs {
     action: BulkActionForConfirmation
   ) => Promise<boolean>;
   showBulkDuplicateConfirmation: () => Promise<string | null>;
+  showManualRuleRunConfirmation: () => Promise<TimeRange | null>;
+  showScheduleBackfillLimitError: () => void;
```

Comment:
Any reason why these two props use different terminology? ManualRun vs Backfill.

If possible I'd keep them unified in the code as well. My personal preference is for "Manual Run" instead of "Backfill" since it seems more self-explanatory by itself, but I understand that Backfill is maybe the technical term - and it's only one word.